deployments/arbitrum/usdc.n/migrations/1686953660_configurate_and_ens.ts

```diff
@@ -0,0 +1,278 @@
+
+    const {
+      bridgeReceiver,
+      timelock: l2Timelock,
+      comet,
+      cometAdmin,
+      configurator,
+      rewards,
+    } = await deploymentManager.getContracts();
+
+    const {
+      arbitrumInbox,
+      arbitrumL1GatewayRouter,
+      timelock,
+      comptrollerV2,
+      governor,
+      USDC,
+      COMP,
+    } = await govDeploymentManager.getContracts();
+
+    const COMPAmountToBridge = exp(12_500, 18);
+    const compGatewayAddress = await arbitrumL1GatewayRouter.getGateway(COMP.address);
+    const refundAddress = l2Timelock.address;
+
+    const compGasParams = await estimateTokenBridge(
+      {
+        token: COMP.address,
+        from: timelock.address,
+        to: rewards.address,
+        amount: COMPAmountToBridge
+      },
+      govDeploymentManager,
+      deploymentManager
+    );
+
+    const configuration = await getConfigurationStruct(deploymentManager);
+
+    const setConfigurationCalldata = await calldata(
+      configurator.populateTransaction.setConfiguration(comet.address, configuration)
+    );
+    const deployAndUpgradeToCalldata = utils.defaultAbiCoder.encode(
+      ['address', 'address'],
+      [configurator.address, comet.address]
+    );
+    const setRewardConfigCalldata = utils.defaultAbiCoder.encode(
+      ['address', 'address'],
+      [comet.address, arbitrumCOMPAddress]
+    );
+    const l2ProposalData = utils.defaultAbiCoder.encode(
+      ['address[]', 'uint256[]', 'string[]', 'bytes[]'],
+      [
+        [configurator.address, cometAdmin.address, rewards.address],
+        [0, 0, 0],
+        [
+          'setConfiguration(address,(address,address,address,address,address,uint64,uint64,uint64,uint64,uint64,uint64,uint64,uint64,uint64,uint64,uint64,uint64,uint104,uint104,uint104,(address,address,uint8,uint64,uint64,uint64,uint128)[]))',
+          'deployAndUpgradeTo(address,address)',
+          'setRewardConfig(address,address)'
+        ],
+        [setConfigurationCalldata, deployAndUpgradeToCalldata, setRewardConfigCalldata]
+      ]
+    );
+
+    const createRetryableTicketGasParams = await estimateL2Transaction(
+      {
+        from: applyL1ToL2Alias(timelock.address),
+        to: bridgeReceiver.address,
+        data: l2ProposalData
+      },
+      deploymentManager
+    );
+
+    const ENSResolver = await govDeploymentManager.existing('ENSResolver', ENSResolverAddress);
+    const subdomainHash = ethers.utils.namehash(ENSSubdomain);
+    const arbitrumChainId = (await deploymentManager.hre.ethers.provider.getNetwork()).chainId.toString();
+    const newMarketObject = { baseSymbol: 'USDC.n', cometAddress: comet.address };
+    const officialMarketsJSON = JSON.parse(await ENSResolver.text(subdomainHash, ENSTextRecordKey));
+    if (officialMarketsJSON[arbitrumChainId]) {
+      officialMarketsJSON[arbitrumChainId].push(newMarketObject);
+    } else {
+      officialMarketsJSON[arbitrumChainId] = [newMarketObject];
+    }
+
+    const mainnetActions = [
```

Comment:
Are we going to seed this deployment with USDC reserves?

Reply:
I think yes, but we need to figure out how to work with CCTP without relying on their attestation api.

Reply:
Yes we should. Discussed with Hans offline on the approach